src/components/singleBolgPage/sections/Header.jsx

```diff
@@ -0,0 +1,14 @@
+// import React from "react";
```

Comment:
is this code meant to be commented? also the directory name is spelled wrong -- singleBlogPage